### DISCUSSION TITLE
KEDA scaledjob not working for Kafka Trigger.

### DISCUSSION THREAD

**Discussion by prtiwari-conga** [kedacore/keda#5305] [Q&A / Need Help]
Hello,

I have a Apache Kafka event driven scaling setup for my application. For 2.1.0 KEDA everything seems to be working fine but as the support for autoscalingv2beta2 (horizontal pod autoscaler) was ending in 1.26 k8 version , i had to update my KEDA to 2.11.2. Since then my Scaledjob is not triggering and when i describe it, i am getting the below error:-

**" KEDAScalerFailed    4m18s (x242 over 44m)  scale-handler  error describing topics: tls: failed to verify certificate: x509: cannot validate certificate for 52.xxx.xxx.xx because it doesn't contain any IP SANs "**

Also my scaledjobs are ready but not active. In my KEDA pod logs i only see one error but do not know what to make of it:-

**ERROR	scaleexecutor	Error setting active condition when triggers are not active	{"scaledJob.Name": "pushinstallconsumerservice-scheduler-scaledjob", "scaledJob.Namespace": "push-upgrades-dev", "error": "client rate limiter Wait returned an error: context canceled"}**

I have tried multiple scenarios now but cannot find a solution for this. Genuine cry for help!!!!

**Comment by JorTurFer** [comment]
Hello @prtiwari-conga 
Have you migrated from 2.1.0 to 2.11.2? 10 versions of difference?
Could you share your ScaledJob and KEDA operator logs?
